### DISCUSSION TITLE
CH2_9 github를 우분투랑 연결을 해서 써야되는건가요?

### DISCUSSION THREAD

**Discussion by HSong94** [tedilabs/fastcampus-devops#61] [Q&A] [answered]
강의를 듣는 동안 파일은  cat > 을 이용해서 제 우분투 루트에 저장을 했었는데 CH2_9에 나오는 app폴더에는 많은 파일들이 있어서 진행하는데 어려움이 있습니다. 혹시 우분투를 github이랑 연결을 해서 써야되는건가요?

**Comment by posquit0** [comment, marked as answer]
안녕하세요~

해당 레포는 패스트캠퍼스 <데브옵스 초격차> 강의 중 제가 진행한 챕터 1, 4, 5, EKS에 대한 질문응답을 위한 공간입니다.
질문 주신 내용이 제 강의 파트가 아니라서 답변이 어려운 점 양해 부탁드립니다.

패스트캠퍼스에 문의를 하시면 담당자 강사 분을 통해 답변을 받을 수 있을 듯 합니다.

감사합니다.

**Reply by HSong94** [reply]
챕터 1이 part 1을 말씀하시는거 맞을까요?

**Reply by posquit0** [reply]
네 맞습니다! 😄 

혹시나 수강중이신 강의가 [초격차 패키지 : 한 번에 끝내는 AWS 인프라 구축과 DevOps 운영](https://fastcampus.co.kr/dev_online_awsdevops) 인지, 패스트캠퍼스의 또 다른 강의인 [Kubernetes 와 Docker로 한 번에 끝내는 컨테이너 기반 MSA](https://fastcampus.co.kr/dev_online_kubemsa) 강의인지도 확인 부탁드립니다.

**Reply by HSong94** [reply]
kubernetes와 docker 강의 입니다

**Reply by posquit0** [reply]
해당 강의는 1강만 제가 진행한 파트입니다. 여러 강사님이 참여한 강의이다보니 혼란이 생긴 것 같습니다!  😭
패스트캠퍼스 문의를 통해 해당 파트 강사님으로부터 답변을 받으실 수 있습니다!

감사합니다. :)

**Reply by HSong94** [reply]
Part 1(1강)의 CH2의 9번째 강의에 대한 질문입니다. 도커 이미지 다루기 - 이미지 빌드 강의입니다. 강사님이 cd를 통해서 3.kub~/3.~ 이렇게 이동하시는거 아는데 제 세팅에는 그 폴더가 없다보니 그렇게 이동하는게 가능하지 않고 또한  git에서 파일 다운이 안되다보니 실습을 진행하지 못하는 중입니다. 그래서 git이랑 연결 같은 것을 해야되는건지에 대한 질문이였습니다

**Reply by posquit0** [reply]
아.. 우선 제가 질문을 제대로 파악하지 못해 죄송합니다~ 이해하였습니다. 다음과 같이 `git clone` 명령어를 이용하여 해당 레포지토리를 로컬 작업 환경에 다운로드 받으신 후에 작업 진행하시면 됩니다! :)
```bash
$ git clone https://github.com/tedilabs/fastcampus-devops
```

**Reply by HSong94** [reply]
감사합니다
